blackbox/docs/sql/reference/create_table.txt

```diff
@@ -313,6 +313,13 @@ Allows to have a read only table.
 :values: Disables the table settings modifications if set to ``true``, if set
          to ``false`` — table settings modifications are enabled.
 
+``mapping.total_fields.limit``
+................................
+
+Sets the maximum number of columns that is allowed for a table. Default is ``1000``.
+
+:value: Number of maximum columns
```

Comment:
as we found out this is not the maximum amount of columns, but columns + internal fields on lucene index